Dumping my code

tillrue opened the pull request:
Here is the code used for my project with short description on what does what.

I could not merge the changes I made into the project yet as they fail the basic "tests" that are run and I haven't figured out yet why that is as the code was working on my computer but didn't pass the "build" process...

tillrue committed:
Create README.md

tillrue committed:
Add files via upload

tillrue committed:
Create README.md

tillrue committed:
Add files via upload

tillrue committed:
Update README.md

tillrue committed:
Update README.md

bryngemark commented:
can you add a link to your PR? then i can help you figure out what's not working. it's probably due to something having been added to trunk since you pulled

bryngemark commented on Wide_Angle_Scattering_Events_Till_Rüber/README.md:
```diff
@@ -0,0 +1,16 @@
+
+3. produceGraph.py - "taskset -c 0,1,2 just fire produceGraph.py 10000 60"
+
+This uses the Analyzer to produce the graphs used in the thesis. It does so for single events like the reconstructed hits in a certain region and also plots for all events like the scoring plane hits. The produced file can be read via a root browser.
```
in the analyzer, you have some branch names that are not standard, like, `...WithBug` etc. maybe i missed it, but i don\t see where/how you set these? 

if someone tries to run the steps you have right now in sequence, i think the code will crash on some input missing. so a more complete instruction would be great.

tillrue replied:
Yes, you're right. It is a "relic" as I have been running the reconstruction multiple times (with changes in the reconstruction code, so bug and stripcenter) and gave the branches different names to be able to plot them at the same time. However, this might seem confusing and I'll delete those from the analyzer.

Instead, I'll describe that the other plots in the report can be obtained through the same analyzer when using the "not fixed" version of reconstructured hits.

bryngemark commented:
> can you add a link to your PR? then i can help you figure out what's not working. it's probably due to something having been added to trunk since you pulled

@tillrue could you point me to the PR? i have trouble finding it myself. it would be great to get your code in before i push too many other changes, that just makes it harder to merge your work.

bryngemark commented:
@tillrue a poke to follow up on my comments + help me help you finish the ldmx-sw PR!

tillrue committed:
Update Analyzer_single_Event.cxx

Graphs for SimpleRecHits_WithBug and _StripCenter have been removed as these require branches of RecHits that are only obtained when the Reconstruction is runned multiple times with the respective modifications in the code.

Instead, the description will be extended to give instructions about how the additional graphs from the report can be obtained.

tillrue committed:
Update README.md

The description was extended to describe how figures 10 and 11 from the report can be obtained.

bryngemark commented on Wide_Angle_Scattering_Events_Till_Rüber/README.md:
```diff
@@ -0,0 +1,16 @@
+I've been running my code using the latest development image (March 2025). It can be run through the following steps:
```
yes, anything that makes it as simple as possible for a newcomer to pick up your work.

bryngemark reviewed (commented):
thanks for working on my comments. just a line about how to set up the image and then i will approve this. and in parallel we should get your PR stated and merged 🙂

tillrue committed:
Update README.md

I have now described in more detail, how "denv" and "just" can be set up as well as how one clones the repository.

tillrue commented:
Alright! I have fixed the description of the code used in the project and will now work on fixing the things on the PR. But I think it should be possible to fix it until the end of the month.
